test/unit/calculators/state_pension_age_calculator_test.rb

```diff
@@ -206,5 +206,57 @@ class StatePensionAgeCalculatorTest < ActiveSupport::TestCase
         end
       end
     end
+
+    context "#claim_can_start_within" do
+      should "return (4 months and 4 days from today)" do
+        Timecop.freeze(Date.parse('2016-01-11')) do
+          calculator = StatePensionAgeCalculator.new({})
+          assert_equal calculator.claim_can_start_within, 125
+        end
+      end
+    end
+
+    context "#approaching_pension_age" do
+      setup do
+        StatePensionAgeCalculator.any_instance.stubs(:claim_can_start_within).returns(127)
```

Comment:
Since we're stubbing time in the tests anyway, I'm not convinced that stubbing`claim_can_start_within` is really helping. I'd be inclined to test the `approaching_pension_age?` method without stubbing anything other than time.